### PR TITLE
Update some presets.

### PR DESCRIPTION
**Pull request by homersimpsons**
A pull request to close a part of my request https://github.com/openstreetmap/iD/issues/3115
I added presets for sop and traffic_signals, and improve a translation (because it was impossible to make the difference beetween zebra crossing and non zebra one).

I edited on web github then I can't run Cmake...

### CODE REVIEW

File: data/presets/presets/tourism/hotel.json [hunk 1]
```diff
@@ -4,7 +4,11 @@
         "operator",
         "address",
         "building_area",
-        "smoking"
+        "smoking",
+        "stars",
+        "rooms",
+        "internet_access"
+        "internet_access/fee"
```
Comment (homersimpsons) [inline data/presets/presets/tourism/hotel.json]: I don't know if this line is correct

**Comment by homersimpsons** [comment]
I added others to match https://github.com/openstreetmap/iD/issues/3144
But again, no cmake

**Comment by bhousel** [comment]
Thanks @homersimpsons!   I cherry picked about half of these commits and also added "rooms", "internet access", "internet access fee" fields to guest_house, hostel, motel presets ..

A few comments:
1. The field name should just be title cased words, no punctuation ("Internet Access Fee" not "Internet access fee ?")
2. You shouldn't update the `fr.json` translation file in the repository here.  These will be available for translation on [Transifex](https://www.transifex.com/ideditor/id-editor/dashboard/) after tonight.  Whatever is saved in the repository gets overwritten by the data pulled down from Transifex when we make a new release of iD.
3. I skipped the traffic signals direction tag because I don't think adding direction to traffic signals is useful.  Or rather - I get that it might be useful when people place the signals not at the junction, but I think it's more likely to confuse people than lead to better data in OSM.
4. I skipped `parallel_direction.json` because I was confused, but I will add it.

Thanks again!  I know you were editing on github - I do encourage you to try to set up a command line environment if you want to do more of this...  It will make things much easier for you!   Try out the instructions in [README](https://github.com/openstreetmap/iD/blob/master/README.md#prerequisites) and let me know if you get stuck..

**Comment by homersimpsons** [comment]
Thanks you

1. Okay, I just looked some example to release those changes then maybe I missinterpreted some values
2. I'm okay but I want to fast correct, this was always annoying because I culdn't distinct zebra and non-zebra crossing preset
3. Okay, as I said in #3115 is it possible do detect if it's an intersection node to display the correct preset?
4. This was to mach cardinal and clock direction name. I didn't find a correct word to explain what is it. Feel free to change this :)

I know a few language and only base (java, html, css, php, python, c++) then I just fix some little issue that I can easily fix without trying to understand the whole code... But maybe later I will come back to help coding when I will get more knowledge about coding.

**Comment by bhousel** [comment]
Great, thanks!

**Pull request closed without merging**